leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,165 @@
+% !TeX root = ../SPL-Leaderboards.tex
+% !TeX spellcheck = en_US
+
+\section{Fastest Walk Leaderboard}
+This leaderboard will measure how fast a NAO will be able to navigate along a both a straight path and one with obstructions.
+
+\subsection{Setup}
+This leaderboard challenge will take place on one half of a standard SPL field, and requires 1 active robot and 4 inactive robots.
+Inactive robots will act as obstacles for half of the attempt and will be placed as follows (illustrated in \cref{fig:walk_leaderboard}):
+\begin{itemize}
+    \item On the penalty mark
+    \item On both corners of the goal area that are not touching the field boundary
+    \item On the corner of the goal area closest to the competing robot
+\end{itemize}
+
+The competing robot will be placed 50cm behind the touchline, in line with the penalty mark.
```

Comment:
One thing missing (same for obstacle) is from where to we take the measurement ? Is it the feet of the robot, the center of the head or anything else ?